resources/views/settings/navbar.blade.php

```diff
@@ -1,4 +1,4 @@
-
+@if($currentUser->can('user-manage') || $currentUser->can('settings-manage') || $currentUser->can('user-roles-manage'))
```

Comment:
Not sure what this achieves because currently if a user doesn't have access to any of these roles, this is what appears,

![screenshot-2017-12-26 user profile bookstack](https://user-images.githubusercontent.com/1685517/34356312-ba44cf3c-ea62-11e7-9cf3-501dcee36b11.png)

Isn't what you are doing the same?